app/lib/features/events/pages/event_page.dart

```diff
@@ -133,19 +132,16 @@ class CalendarEventPage extends ConsumerWidget {
     Set<RsvpStatusTag?> rsvp = <RsvpStatusTag?>{null};
     myRsvpStatus.maybeWhen(
       data: (data) {
-        final status = data.status();
-        if (status != null) {
-          switch (status.tag) {
-            case RsvpStatusTag.Yes:
-              rsvp = <RsvpStatusTag?>{RsvpStatusTag.Yes};
-              break;
-            case RsvpStatusTag.Maybe:
-              rsvp = <RsvpStatusTag?>{RsvpStatusTag.Maybe};
-              break;
-            case RsvpStatusTag.No:
-              rsvp = <RsvpStatusTag?>{RsvpStatusTag.No};
-              break;
-          }
+        switch (data.statusStr(true)) {
+          case 'Yes':
```

Comment:
Any reason for making switch case with hardcoded string instead of using enum?

Reply:
[This](https://github.com/acterglobal/a3/pull/1401#issue-2138226873) is that reason.